cloudhub-2/modules/ROOT/pages/ps-config-advanced.adoc

```diff
@@ -121,7 +121,7 @@ include::partial$select-private-space.adoc[tag=clickAdvanced]
 [[configure-aws-role]]
 == Configure AWS Service Role
 
-If you have identity and access management (IAM) roles configured in AWS, you can associate the role with your private space. The private space receives the permissions from the IAM role in AWS and can access AWS resources. To enable this feature, Anypoint Platform generates a unique role name you can use with AWS IAM. Note that you must allow outbound traffic on port 443 to use this feature. For more information, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html[IAM roles^].
+If you have identity and access management (IAM) roles configured in AWS, you can associate the role with your private space. The private space receives the permissions from the IAM role in AWS and can access AWS resources. To enable this feature, Anypoint Platform generates a unique role name you can use with AWS IAM. To configure this feature, you must use the same organization ID as the organization that originally configured the private space. You must also allow outbound traffic on port 443 to use this feature. For more information, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html[IAM roles^]. 
```

Comment:
configured in/for the ps? 
The organization didn't configure the ps, right?

Reply:
```suggestion
If you have identity and access management (IAM) roles configured in AWS, you can associate the role with your private space. The private space receives the permissions from the IAM role in AWS and can access AWS resources. To enable this feature, Anypoint Platform generates a unique role name you can use with AWS IAM. To configure this feature, you must use the same organization ID as the organization in which private space was originally configured. You must also allow outbound traffic on port 443 to use this feature. For more information, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html[IAM roles^]. 
```

Reply:
Something like:

To configure this feature in AWS:
- Use the unique role that Anypoint Platform generates.
- Use the organization ID originally configured for the private space.
- Allow inbound traffic on port 443 for the private space. (?)

Reply:
should it be "the private space"?